dkim/verify.go

```diff
@@ -246,7 +254,11 @@ func verify(h header, r io.Reader, sigField, sigValue string) (*Verification, er
 	var res *queryResult
 	for _, method := range methods {
 		if query, ok := queryMethods[QueryMethod(method)]; ok {
-			res, err = query(verif.Domain, stripWhitespace(params["s"]))
+			if option != nil && option.LookupTXT != nil {
+				res, err = query(verif.Domain, stripWhitespace(params["s"]), option.LookupTXT)
```

Comment:
I think it'd be fine to just pass the `VerifyOptions` struct to `query`.

Reply:
fixed